tasks/run_cd4pe_job.rb

```diff
@@ -270,7 +270,11 @@ def send_job_output_to_cd4pe(output)
     client = CD4PEClient.new(web_ui_endpoint: api_endpoint, job_token: @job_token, ca_cert_file: @ca_cert_file, logger: @logger)
 
     begin
-      client.make_request(:post, api_endpoint, payload.to_json)
+      response = client.make_request(:post, api_endpoint, payload.to_json)
+      if (response.code != 200)
+        @logger.log("Sending logs directly to CD4PE not supported in CD4PE version < 3.10.0. Printing logs to std out.")
```

Comment:
Nice! Consider making the log message more generic to prevent a possibly incorrect diagnosis of the issue. (We don't know for sure that a non-200 code always means the user is on an older version of CD4PE.)